ludwig/hyperopt/execution.py

```diff
@@ -449,6 +449,10 @@ def _run_experiment(
 
         modified_config = substitute_parameters(copy.deepcopy(hyperopt_dict["config"]), config)
 
+        # Write out the unmerged config to the trial's local directory.
```

Comment:
nit: Can we update this comment to 

```Write out the unmerged config with sampled parameters to the trial's local directory```

Reply:
Updated in [a073cc0](https://github.com/ludwig-ai/ludwig/pull/2640/commits/a073cc00aec3e3915c4cdd143aabbee7a247c816)